server/datastore/mysql/scripts.go

```diff
@@ -294,12 +294,15 @@ func (ds *Datastore) GetHostScriptDetails(ctx context.Context, hostID uint, team
 	}
 
 	var extension string
-	switch hostPlatform {
-	case "darwin":
-		extension = `%.sh`
-		break
-	case "windows":
+	switch {
+	case hostPlatform == "windows":
+		// filter by .ps1 extension
 		extension = `%.ps1`
+	case fleet.IsUnixLike(hostPlatform):
+		// filter by .sh extension
+		extension = `%.sh`
+	default:
+		// no extension filter
```

Comment:
TODO: confirm the expected behavior for other hosts (e.g., chrome)

Reply:
should we hold on reviewing until we get confirmation?

Reply:
This looks correct to me.